From 9a4ae3cf76e0c2d9dec8fcea0ed402bb5a3bc785 Mon Sep 17 00:00:00 2001
From: Rinat Kov <frommrinat@gmail.com>
Date: Tue, 8 Oct 2024 19:57:41 +0200
Subject: [PATCH 8/8] dwdwdwdwd#

---
 src/pages/_app.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pages/_app.js b/src/pages/_app.js
index b97e52f..c9f6183 100644
--- a/src/pages/_app.js
+++ b/src/pages/_app.js
@@ -1,5 +1,5 @@
 import "@/styles/globals.css";
 
 export default function App({ Component, pageProps }) {
-  return <Component {...pageProps} />;
+  return <Component {...pagerops} />;
 }
-- 
2.39.3 (Apple Git-146)

